src/js/providers/tracks-mixin.js

```diff
@@ -295,6 +295,8 @@ define(['../utils/underscore',
         this._unknownCount = 0;
         this._activeCuePosition = null;
         if (this._renderNatively) {
+            // Removing listener first to ensure that removing cues does not trigger it unnecessarily
+            this.removeTracksListener(this.video.textTracks, 'change', this.textTrackChangeHandler);
```

Comment:
@robwalch we have to change the track mode to `hidden` to remove the cues, then set the mode to `disabled` after. This causes up to 2 unnecessary track change events if we don't remove the listener before.